test_runner/fixtures/neon_fixtures.py

```diff
@@ -476,7 +476,7 @@ def init_start(self, initial_tenant_conf: Optional[Dict[str, str]] = None) -> Ne
 
         # Prepare the default branch to start the postgres on later.
         # Pageserver itself does not create tenants and timelines, until started first and asked via HTTP API.
-        log.info(
+        log.debug(
             f"Services started, creating initial tenant {env.initial_tenant} and its initial timeline"
         )
```

Comment:
This is a sneaky one but ... Happy to move it to some other PR :) I doubt anyone will miss it.